Automattic/wp-calypso
# Show VideoPress as "included in plan" for JP Premium and Business features lists

#### Proposed Changes

* add VideoPress plans to the Jetpack Premium and Business features lists so they appear as "included" for the legacy plans

#### Testing Instructions



* Add Jetpack Premium to a jetpack site
* Visit http://calypso.localhost:3000/plans/YOUR-SITE-URL
* Select the Plans tab
* Scroll down until you see VideoPress, it should be labelled as "Part of the current plan"
* Do the same for Jetpack Business

#### Pre-merge Checklist






Related to # 1272-gh-Automattic/greenhouse
